dispatch-rest-delegates/pom.xml

```diff
@@ -87,9 +88,14 @@
 
                 <dependencies>
                     <dependency>
-                        <groupId>com.gwtplatform</groupId>
-                        <artifactId>gwtp-build-tools</artifactId>
-                        <version>${gwtp.version}</version>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>${checkstyle.version}</version>
```

Comment:
It's important to explicitly set checkstyle to 6.2 because the latest plugin defaults to 5.7.